docker-engine.nuspec

```
@@ -6,7 +6,7 @@
     <metadata>
         <!-- == PACKAGE SPECIFIC SECTION == -->
         <id>docker-engine</id>
-        <version>24.0.7</version>
+        <version>24.0.7.1</version>
```

```suggestion
        <version>24.0.7.20231201</version>
```

Use Package Fix Notation: https://docs.chocolatey.org/en-us/create/create-packages#package-fix-version-notation